Test/testNS1.cpp

```diff
@@ -0,0 +1,45 @@
+/* testNS1.cpp continuously reads photodiode data and transmits each data point via serial
+ * Usage:
+ *  This is a quick and dirty script based on the original NS1 software to test the basic functionality of NanoSam I
+ *  Run this script through Teensyduino to test the NS1 payload, monitor via the serial monitor/plot
+ *  
+ */
+
+/* - - - - - - Includes - - - - - - */
+// C++ libraries
+
+// Other libraries
+#include <SPI.h>
+
+/* - - - - - - Initialization - - - - - - */
+const int ADC_CHIP_SELECT = 10; //chip select pin number for ADC
+
+/* - - - - - - Functions - - - - - - */
+
+// getADC reads and returns one 16 bit data point from the ADC via SPI
+uint16_t getADC()
+{
+  uint16_t photodiode16;
+  SPI.beginTransaction(SPISettings(50, MSBFIRST, SPI_MODE3)); //SPISettings(maxSpeed,dataOrder,dataMode)
```

Comment:
Why this dataOrder and dataMode? I have no qualms - just curious what the options are.

Reply:
All three of those arguments are dictated by the slave — the ADC in this case, it may be different for the flash modules. I just copied it over from the old code so I don't know if they got it right. I am particularly doubtful of that maxSpeed since it is usually in the MHz range.